src/main/kotlin/com/kotlindiscord/bot/api/KDCommand.kt

```diff
@@ -7,6 +7,25 @@ import org.apache.commons.text.StringTokenizer
 
 private val logger = KotlinLogging.logger {}
 
+/**
+ * Dataclass representing a command metadata.
+ *
+ * @param name the name of the command
+ * @param help command help text
+ * @param signature command signature
+ */
+data class CommandInfo(val name: String, val help: String, val signature: String) {
```

Comment:
I'm still not 100% sold on using a dataclass for this. What's the reasoning of doing this rather than, say, using attributes on the KDCommand object?